endpoints/openrtb2/video_auction.go

```diff
@@ -306,6 +298,15 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 		return
 	}
 
+	// Populate any "missing" OpenRTB fields with info from other sources, (e.g. HTTP request headers).
+	deps.setFieldsImplicitly(r, bidReqWrapper, account)
+
+	errL = deps.validateRequest(bidReqWrapper, false, false, nil, false)
+	if errortypes.ContainsFatalError(errL) {
+		handleError(&labels, w, errL, &vo, &debugLog)
+		return
```

Comment:
Can you cover this with a test case?

Reply:
added `TestVideoRequestValidationFailed`